tests/Fixer/Symfony/NoExtraConsecutiveBlankLinesFixerTest.php

```diff
@@ -19,6 +19,64 @@
  */
 final class NoExtraConsecutiveBlankLinesFixerTest extends AbstractFixerTestCase
 {
+    public function testInline()
+    {
+        $this->getFixer()->configure(
+            array(
+                'break',
+                'continue',
+                'return',
+                'throw',
+                'use',
+                'curly_brace_open',
+            )
+        );
+
+        $this->doTest(
+            '<?php
+$a = function($e) use ($b) { while(3<1)break; while(3<1)continue; if (true) throw $e; return 1; };
+// test
+');
```

Comment:
@keradus what would be the rule here? It is pretty hard to skip closures blindly